.circleci/config.yml

```diff
@@ -185,7 +185,9 @@ executors:
           POSTGRES_PASSWORD: test
       - image: ledgersmb/ledgersmb_circleci-<< parameters.browser >>
     environment:
+      COA_TESTING: 1
```

Comment:
These were disabled for coverage test runs, because coverage tests already run very long. Maybe it's an idea to keep these disabled on the coverage run? (Same for the DB_TESTING parameter.)

Reply:
Moved each to a specific job